repository_service_tuf/cli/artifact/__init__.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: MIT
+
+from typing import Optional
+
+from click import Context
+
+from repository_service_tuf.cli import click, rstuf
+
+
+@rstuf.group()
+@click.option(
+    "--rstuf-server",
```

Comment:
`--api-url` as we already have rstuf in name in the command
Instead of `rstuf artifact --rstuf-url`, it will look like `rstuf artifact --api-url`

Reply:
Good point! In the ceremony we have `--upload-server`, do you think we should perhaps change this to `--api-url` in the future to align?

Reply:
Issue created https://github.com/repository-service-tuf/repository-service-tuf-cli/issues/347